curriculum/challenges/english/01-responsive-web-design/basic-html-and-html5/contour.html

```diff
@@ -0,0 +1,63 @@
+<html>
+<head>
+<meta charset="utf-8">
+<title>OpenCV.js</title>
+</head>
+<body>
+<h2>OpenCV.js</h2>
+<p id="status">OpenCV.js is loading...</p>
+<div>
+    <div class="inputoutput">
+        <img id="imageSrc" src="bird.jpg"/>
+            <div class="caption">
+                Run example:<input type="button" id="myButton"
+                onclick="contour()" value="Run" disabled=true/>
+            </div>
+    </div>
+        <div class="inputoutput">
+            <canvas id="canvasOutput" >
+            </canvas>
+                <div class="caption">Result
+                </div>
+        </div>
+</div>
+
+<script type="text/javascript">
+
+//Source:
+//https://docs.opencv.org/3.1.0/d3/dc0/group__imgproc__shape.html#ga17ed9f5d79ae97bd4c7cf18403e1689a
+//https://docs.opencv.org/3.1.0/dd/d1a/group__imgproc__feature.html#ga04723e007ed888ddf11d9ba04e2232de
+//https://docs.opencv.org/3.4/d6/d6e/group__imgproc__draw.html#ga746c0625f1781f1ffc9056259103edbc
+//https://docs.opencv.org/3.4/d3/dc0/group__imgproc__shape.html#ga17ed9f5d79ae97bd4c7cf18403e1689a
+
+function contour() {
+    let mat = cv.imread(document.getElementById("imageSrc"));
+    let edged = new cv.Mat.zeros(mat.rows, mat.cols, cv.CV_8UC3);
+    let contours = new cv.MatVector();
+    let hierarchy = new cv.Mat();
+
+    cv.Canny(mat, edged, 100, 200);
+
+    cv.findContours(edged, contours, hierarchy, cv.RETR_CCOMP, cv.CHAIN_APPROX_SIMPLE);
+
+    cv.drawContours(mat, contours, -1, new cv.Scalar(0, 255, 0, 255), 1, cv.LINE_8, hierarchy, 100);
```

Comment:
Can we use only first 4 arguments because the rest of them have default values?
https://docs.opencv.org/master/d6/d6e/group__imgproc__draw.html#ga746c0625f1781f1ffc9056259103edbc